src/analyses/mCP.ml

```diff
@@ -268,9 +268,6 @@ struct
           Result.meet a @@ S.query ctx' q
         in
         match q with
-        | Queries.PrintFullState ->
-          ignore (Pretty.printf "Current State:\n%a\n\n" D.pretty ctx.local);
-          ()
```

Comment:
I'm not sure, but do we want to keep this one around for debugging?

Reply:
It was added in 3bd9caf5621ab7db1d2eb7fba0bda93c161a484b and never used. Also quite impractical and roundabout way to debug by abusing the in-analysis queries instead of just directly printing/tracing.